src/applications/ivc-champva/10-7959C/tests/containers/IntroductionPage.unit.spec.js

```diff
@@ -6,58 +6,58 @@ import { expect } from 'chai';
 import formConfig from '../../config/form';
 import IntroductionPage from '../../containers/IntroductionPage';
 
-const props = {
```

Comment:
Adjusted the props and mockStore to play nice with logged-in state for these tests